ScnSoft-Education/LeetCodeTraining/LongestPalindrome4/LongestPalindromicSubstring.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace LeetCodeTraining.LongestPalindrome
+{
+    //https://leetcode.com/problems/palindrome-number/description/

```

Comment:
The link is still wrong, because this PR about longest palindrome problem, please fix.
Also please add link for LengthOfLongestSubstring problem solving within this PR.

Reply:
Has the second note above been resolved?